src/views/RecIM/views/Admin/index.jsx

```diff
@@ -273,6 +347,188 @@ const Admin = () => {
         </Typography>
         <Typography variant="body1">This action cannot be undone.</Typography>
       </GordonDialogBox>
+      <GordonDialogBox
+        title="Rec-IM Report"
+        open={openRecimReportBox}
+        buttonName="Done"
+        buttonClicked={handleCloseRecimReport}
+      >
+        <Grid container justifyContent="center">
+          <Grid item xs={12} lg={10} xl={12}>
+            <Card elevation={10}>
+              <CardHeader
+                title={
+                  <>
+                    <Typography className={styles.title}>Rec-IM Admin Report</Typography>
+                  </>
+                }
+              />
+              <CardContent>
+                <Card>
+                  <CardContent>
+                    <Typography className={styles.reportSubtitle}>
+                      {'From: ' + (recimReport && new Date(recimReport.StartTime).toLocaleString())}
+                    </Typography>
+                    <Typography className={styles.reportSubtitle}>
+                      {'To: ' + (recimReport && new Date(recimReport.EndTime).toLocaleString())}
+                    </Typography>
+                  </CardContent>
+                </Card>
+                <Card className={styles.reportCard}>
+                  <CardHeader
+                    className={styles.cardHeader}
+                    title={
+                      <Typography className={styles.cardHeader}>
+                        Active Participants:{' '}
+                        {' ' + (recimReport && recimReport.NumberOfActiveParticipants)}
+                      </Typography>
+                    }
+                  />
+                  <CardContent>
+                    {recimReport &&
+                      recimReport.ActiveParticipants.map((participants) => (
```

Comment:
What happens if there are no `ActiveParticipants`?

Reply:
It actually works totally fine, I tested it to be sure, but I think since the JSON still has the Active Participants array, it's just empty, it doesn't cause an issue.